mart/attack/adversary.py

```diff
@@ -76,7 +75,7 @@ def __init__(
             self._attacker = partial(
                 pl.Trainer,
                 num_sanity_val_steps=0,
-                logger=False,
+                logger=list(kwargs.pop("logger", {}).values()),
```

Comment:
I think this should go into a different PR, in part, because, I think it may be better to inherit the logger from the other trainer.

Reply:
Looks like there isn't an easy way to get the base logger.